### PR TITLE
tests: Add Dockerfile for docker integration swarm tests

### LINKED ISSUE #387: add dockerfile for swarm tests
Add dockerfile for nginx for integration swarm tests, this will enable the ip command

### PR DESCRIPTION
**Pull request by GabyCT**
Add Nginx Dockerfile that enables the ip command that will be
used in swarm integration tests

Fixes #387

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by devimc** [comment]
lgtm

**Comment by devimc** [comment]
@mcastelino could you review this PR?

File: Dockerfiles/SwarmTest/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# NGINX image for Intel® Clear Containers swarm tests
+
+In order to build the nginx image which is used in our swarm tests,
```
Comment (iphutch) [inline Dockerfiles/SwarmTest/README.md]: Remove "in order" and "which is":
To build the nginx image used in our swarm tests, follow these steps:

File: Dockerfiles/SwarmTest/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+follow these steps:
+
+1. Build the nginx image with the following command:
```
Comment (iphutch) [inline Dockerfiles/SwarmTest/README.md]: "with the following command" is assumed:
Build the nginx image:

File: Dockerfiles/SwarmTest/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+```
+$ docker build -t $name -f Dockerfile.nginx .
+```
+
+2. Verify the nginx image with the following command:
```
Comment (iphutch) [inline Dockerfiles/SwarmTest/README.md]: Ditto:
Verify the nginx image:

**Comment by GabyCT** [comment]
@iphutch changes applied

File: Dockerfiles/SwarmTest/Dockerfile.nginx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+						nginx=${NGINX_VERSION} \
+						nginx-module-xslt=${NGINX_VERSION} \
+						nginx-module-geoip=${NGINX_VERSION} \
+						nginx-module-image-filter=${NGINX_VERSION} \
+						nginx-module-njs=${NJS_VERSION} \
+						gettext-base \
+	&& rm -rf /var/lib/apt/lists/*
+
+# forward request and error logs to docker log collector
+RUN ln -sf /dev/stdout /var/log/nginx/access.log \
```
Comment (mcastelino) [inline Dockerfiles/SwarmTest/Dockerfile.nginx]: @GabyCT I thought this was a 9p bug was fixed. Do we still need the workaround? I had added this for a lateny 9p issue

**Comment by GabyCT** [comment]
@mcastelino the main issue is that you can not run an "ip a" this is not supported anymore using the Nginx docker hub image however this was actually seen previously in CC 2.1x

**Comment by mcastelino** [comment]
@GabyCT LGTM

**Comment by GabyCT** [comment]
@mcastelino https://github.com/01org/cc-oci-runtime/issues/1013

**Pull request merged**